Automattic/wp-calypso
# Signup/login: update eye icon

Resolves https://github.com/Automattic/wp-calypso/issues/80506

## Proposed Changes

* Update seen/unseen icon at password input for login and signup pages

### Before

<img width="446" alt="Screenshot 2023-08-17 at 11 33 35" src="https://github.com/Automattic/wp-calypso/assets/87168/d624005e-62d5-4448-82fb-1c7e263b1590">
<img width="449" alt="Screenshot 2023-08-17 at 11 33 39" src="https://github.com/Automattic/wp-calypso/assets/87168/e0ceda80-11ff-4894-9a97-756afdcc72ba">

### After

<img width="443" alt="Screenshot 2023-08-17 at 11 33 21" src="https://github.com/Automattic/wp-calypso/assets/87168/6b18ebfa-2d95-471e-9070-b0711617dc3a">
<img width="443" alt="Screenshot 2023-08-17 at 11 33 25" src="https://github.com/Automattic/wp-calypso/assets/87168/634f51e5-2bda-4f62-a6a8-247c5beba036">


## Testing Instructions



Check different login and signup pages. Easiest to start from login pages, and then click "Signup" from there.

Below links are to localhost Calypso, but you can modify them to the Calypso live:

* [Dotcom](http://calypso.localhost:3000/log-in)
* [WooCommerce](http://calypso.localhost:3000/log-in?client_id=50916&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%2F%3Fresponse_type%3Dcode%26client_id%3D50916%26state%3D26b5b2c2efcdd87266c05af98d62c629d9dd6dfb63c1d63fe68db6f265125e91%26redirect_uri%3Dhttps%253A%252F%252Fwoocommerce.com%252Fwc-api%252Fwpcom-signin%253Fnext%253D%25252F%26blog_id%3D0%26wpcom_connect%3D1%26wccom-from%26calypso_env%3Dproduction%26from-calypso%3D1)
* [Jetpack](http://calypso.localhost:3000/log-in?client_id=69040&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fresponse_type%3Dtoken%26client_id%3D69040%26redirect_uri%3Dhttps%253A%252F%252Fcloud.jetpack.com%252Fconnect%252Foauth%252Ftoken%253Fnext%253D%25252F%26scope%3Dglobal%26blog_id%3D0%26from-calypso%3D1)
* [VideoPress](http://calypso.localhost:3000/start/videopress-account/user/en?variationName=videopress&flow=videopress&pageTitle=Video%20Portfolio&redirect_to=/setup/videopress/options)
* [P2](http://calypso.localhost:3000/log-in?redirect_to=http%3A%2F%2Fcalypso.localhost%3A3000%2Fstart%2Fp2%2Fp2-confirm-email%3Fref%3Dp2-lp&from=p2&signup_url=%2Fstart%2Fp2%2Fuser%3Fref%3Dp2-lp)
* Sensei
* ...probably more. 😅 

⚠️ I've also checked all CSS modifying `form-password-input__toggle-visibility` to make sure there weren't any more custom modifications to the icon across flows

The two modifications I'm touching are for VideoPress and P2:

https://github.com/Automattic/wp-calypso/blob/2c27610b13cf13603baa20bd0361db4e4c42d2ec/client/assets/stylesheets/_videopress-extends.scss#L41-L58
https://github.com/Automattic/wp-calypso/blob/2c27610b13cf13603baa20bd0361db4e4c42d2ec/client/assets/stylesheets/_p2-extends.scss#L41-L59


## Pre-merge Checklist
